libs/core/src/lib/button/button.component.ts

```diff
@@ -39,7 +39,7 @@ import { applyCssClass } from '@fundamental-ngx/core/utils';
         '[attr.type]': 'type',
         '[attr.disabled]': '_disabled || null',
         '[attr.aria-label]':
-            'ariaLabel ? ariaLabel : specialButtonType.includes(fdType) ? label != null ?  label +" ,"+ fdType : glyph != undefined || glyph != null ? fdType +","+ glyph.split("-").join(" ")  : title : title'
+            'ariaLabel ? ariaLabel : specialButtonType.includes(fdType) ? label != null ? label : title : title'
```

Comment:
This influences many places.
I saw many accessibility fixes are relying on it.
Why do we need this?

Reply:
Anyway, can be simplified

```
ariaLabel || specialButtonType.includes(fdType) && label || title
```